pexec/managed_process_unix.go

```diff
@@ -126,6 +127,14 @@ func (p *managedProcess) kill() (bool, error) {
 	return forceKilled, nil
 }
 
+// forceKill kills everything in the process group. This will not wait for completion and may result in a zombie process.
```

Comment:
> a zombie process

I have heard this term used in a couple different ways. What do you mean by "zombie process"?

Reply:
```suggestion
// forceKill kills everything in the process group. This will not wait for completion and may result in the kill becoming a zombie process.
```